cql3/restrictions/statement_restrictions.cc

```diff
@@ -1394,6 +1395,94 @@ bool is_satisfied_by(
         }, restr);
 }
 
+/// A column's bound, from WHERE restrictions.
+class bound_t {
+    bool _unbounded;
+    bytes_opt _value; // Invalid when _unbounded is true.
+    const abstract_type& _value_type;
+  public:
```

Comment:
Our coding style indents accessiblity modifiers at the same level as class.

Reply:
Fixed.